public/pages/CreateMonitor/components/MonitorTimeField/utils/validation.test.js

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React from 'react';
+
+import { validateTimeField } from './validation';
+
+describe('validation', () => {
+  test('prompts user to select time field when no value is selected', () => {
+    const dateFields = ['customer_birth_date', 'order_date', 'products.created_on'];
+    const value = undefined;
+    expect(validateTimeField(dateFields)(value)).toBe('Select a time field');
```

Comment:
Small issue, but I believe the UX convention is to end error messages with periods; e.g., `.`.